.github/workflows/create_release.yml

```diff
@@ -1,17 +1,13 @@
 # Create a draft release and build and upload all installers to it.
-name: Create Draft Release
+name: Create Release
 
 on:
   workflow_dispatch:
     inputs:
-      commit_to_release:
-        description: 'Enter commit hash to release (example: ef4037cb571f99cb4919b520fde7174972aae473)'
+      commitish_to_release:
```

Comment:
typo commitish?

Reply:
nope, apparently that's a thing https://git-scm.com/docs/gitglossary